api/_lib/parser.ts

```diff
@@ -35,8 +39,10 @@ export function parseRequest(req: IncomingMessage) {
     fileType: extension === 'jpeg' ? extension : 'png',
     text: decodeURIComponent(text),
     theme: theme === 'dark' ? 'dark' : 'light',
+    backgroundType: backgroundType === 'image' ? 'image' : 'pattern',
```

Comment:
```suggestion
    backgroundType
```

This should be all we need (the shorthand for `backgroundType: backgroundType`). Since background type can _only be_ 'image' or 'pattern', we can just assign it directly without the use of the ternary logic.

Reply:
Hey @zackkrida I tried doing just that, but I am getting the following error:

The type of `backgroundType` is `BackgroundType`, but we assign to it the `backgroundType` received from the destructing `{backgroundType} = query || {}`, hence it's type is `string | undefined`. Thus we need to only assign it when it is a `string. 
Let me know if there is any better method to do this in your mind (Perhaps converting it to a string before assigning it below?)
<img width="533" alt="Screenshot 2021-02-27 at 8 16 16 AM" src="https://user-images.githubusercontent.com/52193408/109373341-2ab77c80-78d4-11eb-929c-e1c98de6b80a.png">

Reply:
We can see the same syntax used for `theme: theme === 'dark' ? 'dark' : 'light',`  Where theme on the left is of the type `Theme` and the one on the right is of the type `string | undefined`

Reply:
@zackkrida I'm waiting for your valuable inputs on this.